internal/impl/sentry/processor_capture.go

```diff
@@ -51,6 +51,13 @@ func newCaptureProcessorConfig() *service.ConfigSpec {
 				Example(`root = {"order": {"product_id": "P93174", "quantity": 5}}`).
 				Example(`root = deleted()`),
 
+			service.NewBloblangField("extras").
+				Version("TBD").
+				Description("A mapping that must evaluate to an object-of-objects. If this mapping produces a value, then it is set on a sentry event as extras.").
+				Optional().
+				Example(`root.foo = "bar"`).
```

Comment:
Is this misleading? The description states it must be an "object of objects" which this does not seem to be that

Reply:
Seems like it just need to be an object from looking at the code (and probably copy/pasted from above). Can we update the description?

Reply:
Fixed